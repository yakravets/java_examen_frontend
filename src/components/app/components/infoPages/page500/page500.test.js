import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import toJson from "enzyme-to-json";

import Page500 from './page500';

it("renders correctly", () => {
    const tree = shallow( < Page500 /> );
    expect(toJson(tree)).toMatchSnapshot();
});

it("renders without crashing", () => {
    shallow( < Page500 /> );
});