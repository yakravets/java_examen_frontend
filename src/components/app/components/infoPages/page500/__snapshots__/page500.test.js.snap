// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Fragment>
  <Result
    extra={
      <Button
        type="primary"
      >
        Back Home
      </Button>
    }
    status="500"
    subTitle="Sorry, something went wrong."
    title="500"
  />
</Fragment>
`;
